packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -221,8 +221,17 @@ class _FlexibleSpaceBarState extends State<FlexibleSpaceBar> {
   }
 }
 
-class _FlexibleSpaceBarSettings extends InheritedWidget {
-  const _FlexibleSpaceBarSettings({
+/// Wraps a widget to convey sizing and opacity information down to the
+/// [FlexibleSpaceBar] contained within the [FlexibleSpaceBar.child].
+///
+/// The [FlexibleSpaceBarSettings.child] widget must contain a
```

Comment:
I don't think this sentence adds any value. Instead I would link to the FlexibleSpaceBar class.